### PR TITLE
Add Bio Stubs for Famous Alumni

### LINKED ISSUE #628: Create people stubs for famous alums
Go through https://en.m.wikipedia.org/wiki/Nottingham_New_Theatre#Alumni, create people stubs for famous alumni filling out careers and other bits.

### PR DESCRIPTION
**Pull request by BenWoodford76**
Have added a person stub for each famous alumni from the wikipedia page.
Have added headshots for those who are easily obtainable
closes #628

### CODE REVIEW

File: _people/daniel_weyman.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Daniel Weyman
+gender: male
+headshot: X6dqbwJ
+submitted: false
+Careers:
```
Comment (wjdp) [inline _people/daniel_weyman.md]: Capital `C` here, need to be lower

Comment (BenWoodford76) [inline _people/daniel_weyman.md]: Will sort :)

**Comment by nhyty** [comment]
Can't see why the build is failing, @wjdp any ideas?

**Comment by wjdp** [comment]
Not looking at Travis logs, will pull later and have a look

**Comment by BenWoodford76** [comment]
Any idea why just this branch is massively failing every time a build is attempted?

**Comment by BenWoodford76** [comment]
Shall try creating a new branch and re-inputting the changes to see if that for some strange reason fixes it?

**Comment by wjdp** [comment]
Haven't had a chance to have look yet. Will do before end of the week.

**Comment by nhyty** [comment]
Any ideas @

**Comment by wjdp** [comment]
Apologies, haven't gotten around to this yet. Will have a look in next few days.

**Comment by BenWoodford76** [comment]
Managed to take a look yet Will?

**Comment by davidtaylorhq** [comment]
There are 4 people in this PR which don't have any show credits associated, see issue #714

**Comment by wjdp** [comment]
Just rebased this against master, fix should apply and build should pass.

**Comment by wjdp** [comment]
Huzah

**Pull request merged**